packages/react-dom/src/events/ReactDOMEventReplaying.js

```diff
@@ -38,10 +38,8 @@ export function setAttemptSynchronousHydration(fn: (fiber: Object) => void) {
   attemptSynchronousHydration = fn;
 }
 
-let attemptDiscreteHydration: (fiber: Object) => void;
-
-export function setAttemptDiscreteHydration(fn: (fiber: Object) => void) {
-  attemptDiscreteHydration = fn;
+export function attemptSynchronousEventHydration(fiber: Object): void {
```

Comment:
nit: `SynchronousEventHydration` makes it sound like the event is what's sync. Maybe `attemptSynchronousHydrationOnFiber` or just `attemptSynchronousHydration`?

Reply:
Yeah I just needed it to be different from `attemptSynchronousHydration` so I added Event in there but I could just add an underscore to the local version of `attemptSynchronousHydration`